### DISCUSSION TITLE
Forget

### DISCUSSION THREAD

**Discussion by Overheadcams** [libre-tube/LibreTube#1418] [Q&A] [answered]
What happens if you forget your account? Can it be restored or does it get deleted after a while or does it sit there and wait for someone to log in to it?

**Comment by Bnyro** [comment, marked as answer]
Accounts don't get deleted directly automatically, but there's no way to restore them when deleted or password lost!

**Comment by Overheadcams** [comment]
dont think i deleted the account tho. I just cant remember the login

**Reply by Bnyro** [reply]
Restoring is not possible either then since you didn't provide any e-mail or something similar when signing up, so there's no way to proof you're the original account creator
